web/secondary_toolbar.js

```diff
@@ -157,6 +174,7 @@ class SecondaryToolbar {
     this._bindCursorToolsListener(options);
     this._bindScrollModeListener(options);
     this._bindSpreadModeListener(options);
+    this._bindReadingDirectionListener(options);
```

Comment:
Please note that you also need add
```css
#secondaryToolbarButtonContainer.hiddenReadingDirectionButtons > .readingDirectionButtons
```
to https://github.com/mozilla/pdf.js/blob/master/web/viewer.css#L638 and then add the appropriate lines in the "baseviewerinit" event listener a few lines below here.